charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-storage-class.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.storageClass.create -}}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ .Values.storageClass.name }}
+  namespace: {{ .Release.Namespace }}
+{{ include "azuredisk.labels" . | indent 2 }}
+provisioner: {{ .Values.driver.name }}
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+allowVolumeExpansion: true
+parameters:
+  cachingmode: None
+  maxShares: "2"
```

Comment:
For now, omit the `cachingmode` and `maxShares` parameter. Not all workloads will need replica mounts and shouldn't have reduced I/O performance by default.

We may want to enable [performance optimizations](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/master/docs/enhancements/feat-add-ability-to-tune-azuredisk-performance-parameters.md) on some of the support storage types, though. We should discuss what defaults we should have with @abhisheksinghbaghel  and Vybava.